apps/core/components/link/index.tsx

```diff
@@ -9,8 +9,10 @@ type LinkType = Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, keyof LinkPr
     children?: React.ReactNode;
   } & React.RefAttributes<HTMLAnchorElement>;
 
+const prefetchDefault = process.env.DEFAULT_PREFETCH_BEHAVIOR === 'true';
```

Comment:
```suggestion
const prefetchDefault = process.env.NEXT_PUBLIC_DEFAULT_PREFETCH_BEHAVIOR === 'true';
```

Or does next remove NEXT_PUBLIC by default? 🤔